docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/block/ncids/block--ncids-header.html.twig

```diff
@@ -21,12 +21,12 @@
           {% endif %}
           <input class="usa-input" id="nci-header-search__field" type="search" name="swKeyword" {{input_attribs}} />
 
-          <button class="usa-button nci-header-search__search-button" type="submit" aria-label="{{ 'Search' | t }}">
-            <span class="nci-header-search__label" aria-hidden="true">{{ 'Search' | t }}</span>
+          <button class="usa-button nci-header-search__search-button" type="submit" aria-label="search">
```

Comment:
This breaks translations. You can't just copy paste here. Please fix this.

Also I thought the <svg> was the problem and I don't see that being updated.